src/lib/spinel/CMakeLists.txt

```diff
@@ -61,7 +61,6 @@ set(COMMON_INCLUDES
 )
 
 set(COMMON_SOURCES
-    radio_spinel.cpp
```

Comment:
I agree on this as well. We don't want `radio_spinel` as part of general purpose `spinel` library.